actionpack/test/dispatch/resource_isolation_policy_test.rb

```diff
@@ -245,6 +246,7 @@ def initialize(app)
       def call(env)
         env["action_dispatch.resource_isolation_policy"] = ActionDispatch::ResourceIsolationPolicy.new do |policy|
           policy.same_site = false
+          policy.log_warning_on_fetch_metadata_failure = false
```

Comment:
Why do you need to change the default `true` for these tests?

Reply:
Otherwise, the tests start spamming STDOUT.